docs/services/gracc-corrections.md

```diff
@@ -0,0 +1,62 @@
+Installing GRACC Corrections
+============================
+
+GRACC Corrections are used to modify records during the summarization process.  RAW records are not modified in the correction process.  The correction is applied after summarization and aggregation, but before the record is enriched with data from [Topology](https://topology.opensciencegrid.org/).
```

Comment:
I think Brian has a rule about requiring linebreaks after each sentence / period

Reply:
Yeah, it makes diffs cleaner in the future but I don't own the Operations repo so the conventions are up to @jdost321